airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -113,6 +113,11 @@ def is_oauth2(self) -> bool:
 
         return credentials_title == "OAuth Credentials"
 
+    def is_private_app(self) -> bool:
+        credentials_title = self.credentials.get("credentials_title")
+
+        return credentials_title == "Private App Credentials"
```

Comment:
I was looking for this feature and glad that I found a PR _hot of the press_

I was interested about the code change required for this feature and while looking at the code I thought I could do this very small suggestion, i hope you don't mind. There already was a comment about using a constant for the hardcoded strings. It's been replaced at one location, but here we find a similar situation.

Alternatively we could move the constants to a `constants.py` file and import the constants at both locations where they are needed. Also indicates that there is a relation between the two files.

Thanks for all effort so far, looking forward to getting this merged!